pyproject.toml

```diff
@@ -17,15 +17,15 @@ dependencies = [
     "nslsii",
     "pyepics",
     "aioca",
-    "pydantic<2.0",
+    "pydantic<2.0",      # Pinned by our required fastapi version, leave pinned until can check incompatibility
     "stomp-py",
     "aiohttp",
     "PyYAML",
-    "click<8.1.4",
+    "click",
```

Comment:
nit: why click no longer pinned?

Reply:
Click was pinned for an issue with private methods that is no longer relevant in the latest version, I've checked and we're able to now unpin it.